addons/udes_stock/models/stock_move.py

```diff
@@ -654,3 +654,40 @@ def _refactor_action_batch_pickings_by(self, by_key):
 
             # Associate picking to the batch.
             picking.write({"batch_id": batch.id})
+
+    def refactor_action_by_maximum_quantity(self):
+        """Split moves to have a maximum quantity"""
+        picking_type = self.mapped("picking_type_id")
+        picking_type.ensure_one()
+
+        if not picking_type.u_assign_refactor_constraint_value:
+            return
```

Comment:
Perhaps we should return self here if we don't do anything.

Reply:
This called on line 271. The expectation in this function is that if one of the `refactor_action` methods returns nothing then nothing has been refactored and moves won't be updated. I could return `self` here and it would still work but there would be some extra logic on line 273-275 triggered; removing self from a recordset and then readding it